amundsen_application/static/js/components/common/ResourceListItem/TableListItem/index.tsx

```diff
@@ -38,7 +39,7 @@ class TableListItem extends React.Component<TableListItemProps, {}> {
             <div className="resource-info-text my-auto">
               <div className="resource-name title-2">
                 <div className="truncated">
-                  { `${table.schema}.${table.name}`}
+                  <SchemaInfo schema={ table.schema } desc={ table.schema_description }placement="bottom"/>.{ table.name }
```

Comment:
We won't want to render a `SchemaInfo` if there is no `schema_description`. I would suggest:
1. If there is not `schema_description` return existing code -- `${table.schema}.${table.name}`.
2. Else render a `SchemaInfo`. In this case I think `SchemaInfo` should take `table.name` as a parameter as well, and own rendering the entire text.

Reply:
In my current implementation, `SchemaInfo` checks if `schema_description` exists or not and returns text accordingly. I will change this as per above suggestion.

Reply:
Actually you really could so it either way. I'd say the main concept here is having logic consolidated into the appropriate chunks. If you want `SchemaInfo` to own the logic for rendering either case, that is also fine. 

This is one if the reason I suggested `SchemaInfo` would take `table.name` as well. So you can also maintain your current implementation but address my 2nd comment.

Reply:
I changed the as per your suggestion, I think by doing this logic is more clear and understandable. Thanks for your suggestion.